implementations/DanubeTech.json

```
@@ -4,6 +4,25 @@
   "issuers": [{
     "id": "did:key:z6MksvTdeczd92YQ5W2V2gL9kXdRvD2AEiyJPoWieP95HUDo",
     "endpoint": "https://uniissuer.io/1.0/credentials/issue",
+    "options": {
+      "type": "Ed25519Signature2018"
+    },
+    "tags": ["VC-API"]
+  },
+  {
+    "id": "did:key:z6MksvTdeczd92YQ5W2V2gL9kXdRvD2AEiyJPoWieP95HUDo",
+    "endpoint": "https://uniissuer.io/1.0/credentials/issue",
+    "options": {
+      "type": "Ed25519Signature2020"
+    },
+    "tags": ["VC-API"]
```

This should also include  the "Ed25519Signature2020" tag. 
```suggestion
    "tags": ["VC-API", "Ed25519Signature2020"]
```